src/plugins/mf_priority.cpp

```diff
@@ -544,18 +544,11 @@ static int priority_cb (flux_plugin_t *p,
                                                     FLUX_JOBTAP_CURRENT_JOB,
                                                     "mf_priority:bank_info"));
 
-    if (b == NULL) {
-        flux_jobtap_raise_exception (p, FLUX_JOBTAP_CURRENT_JOB, "mf_priority",
-                                     0, "internal error: bank info is missing");
-
-        return -1;
-    }
-
-    if (b->max_run_jobs == BANK_INFO_MISSING) {
-        // the association that this job is submitted under could not be found
-        // in the plugin's internal map when the job was first submitted and is
-        // held in PRIORITY by assigning special temporary values to the job;
-        // try to look up the association for this job again
+    if (b == nullptr || b->max_run_jobs == BANK_INFO_MISSING) {
+        // The flux-accounting information associated with this job could not
+        // be found by the time this job got to job.state.priority. Attempt to
+        // look up the association again and attach its information to the job
+        // with aux_set
```

Comment:
Suggestion: Amend the comment to indicate why the aux item may not be set or the bank info is missing. This may help a future developer. E.g. "This could be due to either ..."

Reply:
Good suggestion, thanks! I've just force-pushed up an amendment to this comment as well as referenced RFC 21 for the job state diagram; the image in that RFC is super helpful.